website/contributing/release-testing.md

```diff
@@ -74,8 +75,21 @@ Followed by the wanted options:
   -t, --target      [choices: "RNTester", "RNTestProject"] [default: "RNTester"]
   -p, --platform                    [choices: "iOS", "Android"] [default: "iOS"]
   -h, --hermes                                         [boolean] [default: true]
+  -c, --circleciToken                                                   [string]
 ```
 
+Using the `-c <your-token>` option is recommended, as the script will download the required artifacts from CircleCI, instead of having to build from source reducing the testing time
```

Comment:
```suggestion
Using the `-c <your-token>` option is recommended, as the script will download the required artifacts from CircleCI, instead of having to build from source reducing the testing time.
```